crmsh/parallax.py

```diff
@@ -14,10 +15,11 @@ class Parallax(object):
     # slurp: Copies files from a set of remote hosts to local folders
     """
     def __init__(self, nodes, cmd=None, localdir=None, filename=None,
-                 src=None, dst=None, askpass=False, ssh_options=None):
+                 src=None, dst=None, askpass=False, ssh_options=None, use_crmsh_key=None):
```

Comment:
For now, this option `use_crmsh_key` is only used by behave, which will set the value `False`

Reply:
@liangxin1300 I don't really like the idea. Why do you need this behaviour? If it's because we need the latest version of parallax, don't worry, we will merge it before this, and we should specify that this version needs the latest paralallax version in the `setup.py` of `spec` file

Reply:
By the way, we could avoid this if we add the `ssh_key` option as `kwargs` in the last paralllax change. What do you think about this?

Reply:
For now, behave in crmsh testing always running on one node `hanode1`,
while still need to run command on remote node using parallax like:
`When    Run "crm cluster join -c hanode1 -y" on "hanode2"`,
here parallax should use default ssh key provided by my docker image, not crmsh ssh key

Reply:
I don't get it. I'm afraid this code is hiding some behavioural change. I want to test what happens in the real code, not something done for testing. This is a bad practice in my opinion.
If `parallax` needs to use the default key for joining, just implement this, not this `test mode` code.
To be honest, I don't want this kind of code.